repos/roslyn-tools.proj

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildCommand>$(ProjectDirectory)\build$(ShellExtension) $(FlagParameterPrefix)pack $(FlagParameterPrefix)configuration $(Configuration)</BuildCommand>
+    <!-- TODO: remove /warnaserror:0 (https://github.com/dotnet/arcade/issues/1373) -->
+    <BuildCommand>$(ProjectDirectory)\build$(ShellExtension) $(FlagParameterPrefix)pack $(FlagParameterPrefix)configuration $(Configuration) /warnaserror:0</BuildCommand>
```

Comment:
We have a targeted patch in `release/2.1` for this: https://github.com/dotnet/source-build/blob/release/2.1/patches/roslyn-tools/0001-Ignore-NU5125-warning-error-for-2.1.500-SDK.patch. (https://github.com/dotnet/source-build/commit/e26de38dc987ddc7e1b00747e0c88850c238d3ad updated 2.1 to the 2.1.500 SDK.)

Those changes haven't flowed through to `master` yet. You might actually want to wait for that instead of redoing this work, sorry I didn't catch that sooner.